src/ImageSharp/Processing/Processors/Transforms/Resize/ResizeProcessor.cs

```diff
@@ -26,28 +26,28 @@ public ResizeProcessor(IResampler sampler, int width, int height, Size sourceSiz
         {
             Guard.NotNull(sampler, nameof(sampler));
 
-            // Ensure size is populated across both dimensions.
+            // Ensure target size is populated across both dimensions.
             // If only one of the incoming dimensions is 0, it will be modified here to maintain aspect ratio.
             // If it is not possible to keep aspect ratio, make sure at least the minimum is is kept.
-            const int min = 1;
+            const int Min = 1;
             if (width == 0 && height > 0)
```

Comment:
I wonder if we share code with the other method.

Reply:
I've given the code another once over to remove duplication and respond to comments. Much cleaner now!